python/lsst/analysis/tools/tasks/associatedSourcesTractAnalysis.py

```diff
@@ -70,11 +74,38 @@ class AssociatedSourcesTractAnalysisConnections(
         dimensions=("instrument",),
         isCalibration=True,
     )
+    properMotionCatalogs = ct.Input(
+        doc="Catalog containing proper motions.",
+        name="gbdesAstrometricFit_starCatalog",
```

Comment:
No, I don't think the source tables will ever have proper motion and parallax columns. Soonish there should be some sort of Calibration Objects table that will have these values, but that doesn't exist yet.

Reply:
I think this is worth pushing up to whomever is in charge of the SDM schema. I would expect that our own calculation of PM/parallax should be in our output object tables, since it is something we've said we would report. You can't fix it here, obviously, but we should be prepared to report it there eventually.

Reply:
There is already a long term plan for pm/parallax in the object tables, I just don't think it's going in the source tables.